cpp/test/DataStorm/reliability/test.py

```diff
@@ -8,55 +8,42 @@
     "DataStorm.Trace.Session": 3,
     "DataStorm.Trace.Data": 2,
     "Ice.Trace.Protocol": 1,
+    "Ice.Trace.Network": 3,
 }
 
-# Disable client and server idle timeout below to avoid a test to recover after the connection is closed by the idle timeout.
-# This ensure that test hangs if the session goes into an invalid state, and is not silently recover after the idle timeout
-# closes the connection.
-
 # A client connected to the default server
 clientProps = {
     "DataStorm.Node.Multicast.Enabled": 0,
     "DataStorm.Node.Server.Enabled": 0,
     "DataStorm.Node.ConnectTo": "tcp -p {port1}",
-    "Ice.Connection.Server.IdleTimeout": 0,
```

Comment:
The IdleTimeout settings are no longer required, this is now done programmatically see #4619